### ISSUE TITLE
check for db_password_alias before replacing

### ISSUE THREAD

**Issue by donsizemore** [GlobalDataverseCommunityConsortium/dataverse-ansible#99]
https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/master/tasks/dataverse-install.yml#L73 is no longer needed once https://github.com/IQSS/dataverse/pull/7422 is merged. only replace for older installs.
